Examples/src/main/kotlin/com/mapbox/vision/examples/ArCustomizationActivityKt.kt

```diff
@@ -0,0 +1,50 @@
+package com.mapbox.vision.examples
+
+import com.mapbox.vision.ar.FenceVisualParams
+import com.mapbox.vision.ar.LaneVisualParams
+import com.mapbox.vision.ar.VisionArManager
+import com.mapbox.vision.ar.core.models.Color
+import com.mapbox.vision.ar.view.gl.VisionArView
+
+class ArCustomizationActivityKt : ArActivityKt() {
+
+    override fun setArRenderOptions(visionArView: VisionArView) {
+        // Make sure lane is visible
+        visionArView.setLaneVisible(true)
+        // Set the desired lane length in meters.
+        VisionArManager.setLaneLength(40f)
+
+        // Create an instance of LaneVisualParams
+        val laneVisualParams = LaneVisualParams(
+            // Set the color
```

Comment:
imo `javadocs` for params of  `LaneVisualParams ` should be enough, don't need one more comment

Reply:
I've actually copied iOS code. So will leave just to be consistent. Moreover javadocs are not available via `aar`-s.